src/House.java

```diff
@@ -0,0 +1,13 @@
+/**
+ * Created by sparatan117 on 11/28/16.
+ */
+//stores location, color, house number, price of house, and if it is for rent or not.
+public class House {
+    String location;
+    String outsideColor;
+    int houseNumber;
+    int priceOfHouse;
```

Comment:
Given that this as an attribute of `House` it is redundant to have the word house in the variable.  `price` must refer to house.